File: mango-win-client/Mango-InternalClient/Mango-InternalClient/Models/LoginModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Mango_InternalClient.Models
{
    public class LoginModel
    {
        public string username;
        public string password;
    }

    public class LoginModelResp
    {
        public string message;
        public string ext_user;
        public string ext_user_id;
        public string access_token;
        public string refresh_token;
    }
}

File: mango_flutter_app/lib/models/json/loginModel.dart
class LoginModel {
  final String message;
  final String ext_user;
  final int ext_user_id;
  final String access_token;
  final String refresh_token;

  LoginModel(this.message, this.ext_user, this.ext_user_id,
      this.access_token, this.refresh_token);

  LoginModel.fromJson(Map<String, dynamic> json) :
    message = json['message'],
    ext_user = json['ext_user'],
    ext_user_id = json['ext_user_id'],
    access_token = json['access_token'],
    refresh_token = json['refresh_token'];

  Map<String, dynamic> toJson() =>
      {
        'message' : message,
        'ext_user' : ext_user,
        'ext_user_id' : ext_user_id,
        'access_token' : access_token,
        'refresh_token' : refresh_token
      };
}